simplex/src/test/java/io/em2m/simplex/ConditionTest.kt

```diff
@@ -58,4 +56,51 @@ class ConditionTest : Assert() {
         assertTrue(condition.test("true", "true"))
     }
 
+    @Test
+    fun dateEquals() {
+        val condition = DateEquals()
+        val date = Date()
+        assert(condition.test(date, date))
```

Comment:
Can you add a test here using strings?  Like ```condition.test("2015-04-20", "2015-04-20")```?  We should also test for a negative.